baseplate/server/__init__.py

```diff
@@ -246,13 +261,20 @@ def load_app_and_run_server() -> None:
     # clean up leftovers from initialization before we get into requests
     gc.collect()
 
-    logger.info("Listening on %s, PID:%s", listener.getsockname(), os.getpid())
+    logger.info("Listening on %s", listener.getsockname())
     server.start()
     try:
         shutdown_event.wait()
-        logger.info("Finally stopping server, PID:%s", os.getpid())
+
+        SERVER_STATE.shutting_down = True
+
+        if cfg.drain_time:
```

Comment:
IMO this should be `if cfg.drain_time and cfg.drain_time != 0`, so that we can update our cookiecutter INI file to include the field with a explanatory comment. something like:

```
# drain_time should be greater than k8s' (failureThreshold * periodSeconds)
# see <<insert path to k8s config>>
drain_time = 0
```

Reply:
we can also just have 

```
# drain time should be blah blah...
drain_time =
```

in the config and this'll work as-is

Reply:
(or put a value in the cookiecutter that works for our infra and be done with it :)